src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -78,15 +79,10 @@
 (def mq-addr "vm://localhost?jms.prefetchPolicy.all=1&create=false")
 (def mq-endpoint "puppetlabs.puppetdb.commands")
 
-(defn send-command!
-  "Send the command using the default MQ address and endpoint"
-  [command version payload]
-  (command/enqueue-command! mq-addr mq-endpoint command version payload))
-
 (defn auto-deactivate-nodes!
```

Comment:
looks like you can just change it to #(auto-deactivate-nodes! node-ttl % mq-connection) and everything will work.